tests/acceptance/test_dcos_zookeeper.py

```diff
@@ -24,3 +24,8 @@ def test_delete_zk_node():
     uninstall_package_and_wait('marathon')
 
     assert delete_zk_node('universe/marathon-user')
+
+
+def get_zk_node_data():
+    install_package_and_wait('marathon')
+    get_zk_node_data('universe/marathon-user')
```

Comment:
1. I'll add the `test_` prefix.  Missed that.
2. All the function guarantees is that the server will return a valid znode JSON.  Not any particular JSON.  If the request fails or it's not JSON, the test will fail.